scripts/deploy/DeployContract.ts

```diff
@@ -56,6 +56,64 @@ const libraries = new Map<string, string[]>([
   ['KyberDebtLib', ['KyberLib', 'ConverterStrategyBaseLib2', 'BorrowLib', 'PairBasedStrategyLogicLib']],
 ]);
 
+export async function deployContractSilently<T extends ContractFactory>(
```

Comment:
maybe just add a parameter to the existing function?

Reply:
Its hard because we cant add to end where ...args params located.
If we put params not to the end then need to rewrite all tests and scrupts where deployContract used.